CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 # UNRELEASED
 
 - fix(eth): always return nil for eth transactions not found ([filecoin-project/lotus#12999](https://github.com/filecoin-project/lotus/pull/12999))
+- feat: add experimental v2 APIs that are "F3 aware."  (TODO: expand this section significantly to cover where someone learns about the new APIs, how they enable them, and what expectations they should have around them—i.e., they may change)
```

Comment:
also TODO, when we revisit this, we need the link to this PR; I'll try and remember to add it when I follow-up with Eth stuff if it's not done already by then

Reply:
Help Me ~Obi-Wan Kenobi~ Generative AI, You’re My Only Hope.